.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,196 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI Workflow
+
+on:
+  pull_request_target:
+    branches:
+      - master
+env:
+  SELENIUM_GRID_PROVIDER: saucelabs
+  CLOUD_WATCH_METRIC: false
```

Comment:
I guess this will be turned on once testing succeeds here?

Reply:
No need. This is mainly for canary for monitoring.